src/components/userProfile/UserProfile.js

```diff
@@ -43,7 +43,9 @@ class UserProfile extends Component {
       width: 0,
       height: 0,
       opacity: new Animated.Value(0),
+      activeNaviBar: false,
```

Comment:
I think we need to keep the consistency of our code base. Now, some variable has name Nav, Navbar, NavigationBar, Btn, Button ... and so on. Also, If variable name will not getting too much longer, It is considered as a good practice that just not use a short variable name.

Reference:
http://cc2e.com/Page.aspx?hid=225